portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,9 +49,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
+      /* Convert the comment's timestamp (ms since Epoch) to a date (d/m/y) */
+      LocalDate commentDate = Instant.ofEpochMilli(
```

Comment:
Commented this below but it might be easier to use something like SimpleDateFormat. Here's an example from StackExchange: https://stackoverflow.com/questions/17432735/convert-unix-time-stamp-to-date-in-java